xml/docu_styleguide.language.xml

```diff
@@ -259,11 +259,12 @@
    </listitem>
    <listitem>
     <para>
-     Use a trailing slash to distinguish between directories (for example,
-     <filename>/etc/YaST2/</filename>) and files (for example,
-     <filename>/etc/YaST2/control.xml</filename>).
-     If the distinction is important in the current context, in
-     addition, comment on it in the running text.
+     For directories, add a trailing slash to distinguish between a directory
+     (for example, <filename>/etc/YaST2/</filename>) and a file (for example,
+     <filename>/etc/YaST2/control.xml</filename>). For less experienced Linux
```

Comment:
```suggestion
     <filename>/etc/YaST2/control.xml</filename>). For less-experienced Linux
```

Reply:
Why would you hyphenate this? (mostly just curious)

Reply:
The adjective here is the phrase "less experienced". The test is this: without a hyphen, there are 2 separate adjectives: "less" and "experienced". Do these make sense independently? "Less Linux users" and "experienced Linux users". Well, the latter does, but is the reverse of the intended meaning. So, the adjective is the _pair_ of words — a compound adjective — therefore they should be joined with a hyphen.

Another example... compare: 

The five-year-old girl. ← correct: the adjective here is the phrase "five-year-old".
The girl was five years old. ← correct: not an adjective.
As opposed to:
The girl was five-years-old. ← wrong; not an adjective. (But you will often see this.)

https://www.grammarbook.com/punctuation/hyphens.asp

Reply:
Uhm, there actually is an adverb ("less") and an adjective ("experienced") and we are using the adverb to define the adjective.

Reply:
Yeah, "less" is one of these strange words where it's hard to tell if it's an adverb or an adjective, and that's at least partly why I asked the question. I suspect @sknorr may be right here, that it's an adverb modifying an adjective, rather than being part of a compound adjective, which would explain why the hyphen seemed odd to me.

Reply:
> we are using the adverb to add nuance to the adjective.

I disagree. 

My reasoning is this: if we consider the 2 words independently, then the potential phrases here are:
[1] "for less Linux users" ← basically meaningless in context
[2] "for experienced Linux users". ← the opposite of our intention.

"Less" is not adding nuance here; it is reversing the meaning of the phrase. That is not a subtle shading of meaning; "less" is not qualifying "experienced", we are saying "users who are less experienced". As such, I feel that it is a compound adjective, and compound adjectives should be hyphenated.

I agree, less *can* be an adverb, absolutely, but I think that whether it's one in this instance is irrelevant.

But it's not a big deal and if we continue as current written then it's not a very important thing. The meaning is clear either way.

Reply:
After a discussion in the styleguide meeting today, we decided to for a writing without hyphen, so I'm not applying this suggestion.